tests/fake_texture.rs

```diff
@@ -0,0 +1,18 @@
+extern crate graphics;
+
+use self::graphics::ImageSize;
+
+pub struct FakeTexture {
```

Comment:
Why cant you just do `pub struct FakeTexture;` and initialize it with `FakeTexture`?

so `new` will be `pub fn new() -> Self { FakeTexture }`